cli/spec/fixtures/kontena-with-env-file.yml

```diff
@@ -5,7 +5,7 @@ wordpress:
   stateful: true
   env_file: .env
   environment:
-    WORDPRESS_DB_PASSWORD: ${project}_secret
+    WORDPRESS_DB_PASSWORD: %{project}_secret
```

Comment:
This is only used by the `app` specs, for stacks there's stack-with-env-file.yml. 

I modified the wrong file and used % instead of $ when reverting like it was in another yml.